frontend/src/app/patients/ManagePatients.tsx

```diff
@@ -210,7 +210,9 @@ export const DetachedManagePatients = ({
             patient.isDeleted && "sr-patient-row--removed"
           )}
         >
-          <th scope="row">{editUserLink}</th>
+          <th scope="row" className="sr-patient-row-header">
```

Comment:
is this a row header? or should the reusable class be renamed to .sr-patient-row to describe better the scope of the style?

Reply:
yes! it is a row header.